src/app/components/student-record-display/student-record-display.component.html

```diff
@@ -58,7 +58,7 @@ <h5 class="card-titile d-inline mx-2">Request History</h5>
                                 <table class="table table-striped">
                                     <thead>
                                         <tr>
-                                            <th scope="col">Token Option Name</th>
+                                            <th scope="col">Request&rsquo;s Name</th>
```

Comment:
Not sure should we use "Request's name" here. The request itself is not named, but the token option that is being requested does. Since we need to have the concept of token option for instructors, maybe something like "Requested Token Option" is a better choice.